api/container/containerv2/clusters_test.go

```diff
@@ -46,7 +46,8 @@ var _ = Describe("Clusters", func() {
               "State": "normal",
               "IsPaid": false,
               "IsTrusted": true,
-              "WorkerCount": 1
+              "WorkerCount": 1,
+			  "securityGroupIDs": "null"
```

Comment:
`securityGroupIDs` is not added to `ClusterInfo`, nor present in the `getCluster` swagger:
https://containers.cloud.ibm.com/global/swagger-global-api/#/v2/getCluster

did I miss something?

Reply:
my bad, I removed the unnecessary parameter